src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseClaimantRepReplyService.java

```diff
@@ -0,0 +1,366 @@
+                LINK_TO_CITIZEN_HUB, emailService.getCitizenCaseLink(caseDetails.getCaseId()));
+        respondentEmailAddressList.forEach(respondentEmail ->
+            emailService.sendEmail(replyToTribunalEmailToClaimantTemplateId, respondentEmail, personalisation));
+    }
+
+    private void sendAcknowledgementEmailToLR(CaseDetails caseDetails, String userToken,
+                                              boolean isRespondingToTribunal) {
+        emailService.sendEmail(
+                getAckEmailTemplateId(caseDetails, isRespondingToTribunal),
+                userIdamService.getUserDetails(userToken).getEmail(),
+                TseHelper.getPersonalisationForAcknowledgement(
+                        caseDetails, emailService.getExuiCaseLink(caseDetails.getCaseId())));
+    }
+
+    /**
+     * Send emails when LR submits response to application.
+     */
+    public void sendRespondingToApplicationEmails(CaseDetails caseDetails, String userToken) {
+        // should send the email to respondent
+        sendEmailToRespondentForRespondingToApp(caseDetails);
+        // should send the email to claimant rep
+        sendAcknowledgementEmailToLR(caseDetails, userToken, false);
+        // send the email to tribunal
+        claimantTseService.sendAdminEmail(caseDetails);
+    }
+
+    private String getAckEmailTemplateId(CaseDetails caseDetails, boolean isRespondingToTribunal) {
+        boolean copyToOtherParty = YES.equals(caseDetails.getCaseData().getTseResponseCopyToOtherParty());
+
+        if (isRespondingToTribunal) {
+            return copyToOtherParty
+                    ? replyToTribunalAckEmailToLRRule92YesTemplateId
+                    : replyToTribunalAckEmailToLRRule92NoTemplateId;
+        }
+
+        return copyToOtherParty
+                ? acknowledgementRule92YesEmailTemplateId
+                : acknowledgementRule92NoEmailTemplateId;
+    }
+
+    private void sendEmailToRespondentForRespondingToApp(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        if (!YES.equals(caseData.getTseResponseCopyToOtherParty())) {
+            return;
+        }
+
+        List<String> respondentEmailAddressList = getRespondentEmailAddressList(caseData);
+        if (respondentEmailAddressList.isEmpty()) {
+            return;
+        }
+
+        try {
+            byte[] bytes = tornadoService.generateEventDocumentBytes(caseData, "",
+                    TSE_CLAIMANT_REP_REPLY);
+
+            boolean isWelsh = featureToggleService.isWelshEnabled()
+                    && Optional.ofNullable(caseData.getClaimantHearingPreference())
+                    .map(preference -> WELSH_LANGUAGE.equals(preference.getContactLanguage()))
+                    .orElse(false);
+
+            Map<String, Object> personalisation = TseHelper.getPersonalisationForResponse(caseDetails,
+                    bytes, emailService.getCitizenCaseLink(caseDetails.getCaseId()), isWelsh);
```

Comment:
Respondent rep should be getting ExUI link not citizen hub

Reply:
addressed in [088840c](https://github.com/hmcts/et-ccd-callbacks/pull/2259/commits/088840cf6cb992ae2b002e69019b5941c899201b)